File: Wei.Core/Domain/Sys/Property.cs
using System.Collections;
using System.Collections.Generic;

namespace Wei.Core.Domain.Sys
{
    public class Property : BaseEntity
    {
        private ICollection<PropertyValue> _propertyValueList;

        public string PCode { get; set; }
        public string Desc { get; set; }
        public int SelectValueType { get; set; }

        public virtual ICollection<PropertyValue> PropertyValueList
        {
            get { return _propertyValueList ?? (_propertyValueList = new List<PropertyValue>()); }
            set { _propertyValueList = value; }
        }
    }
}

File: Wei.Web/app/model/sys/Property.js
Ext.define('Wei.model.sys.Property', {
    extend: 'Wei.model.Base',

    fields: [
        { type: 'int', name: 'id', persist: true },
        { type: 'string', name: 'pcode', persist: false },
        { type: 'string', name: 'desc', persist: false },
        { type: 'int', name: 'selectvaluetype', persist: true }

    ]
});
